src/how-to/render/display-text.md

```diff
@@ -1,5 +1,164 @@
 # Displaying Text
 
-(Stub) This page covers how text displaying works. It will cover `Line`, `Span`, `Text`, the
-migration of `Spans` to `Text`, and how these can be created, styled, displayed, altered, and such.
-These are all interrelated topics so, they are on the same page.
+This page covers how text displaying works. It will cover `Span`, `Line`, and `Text`, and how these
+can be created, styled, displayed, altered, and such.
+
+## `Span`
+
+A `Span` is a styled segment of text. You can think of it as a substring with its own unique style.
+It is the most basic unit of displaying text in `ratatui`.
+
+The examples below assume the following imports:
+
+```rust
+use ratatui::{prelude::*, widgets::*};
+pub type Frame<'a> = ratatui::Frame<'a, CrosstermBackend<std::io::Stderr>>;
+```
+
+A `Span` consists of "content" and a "style" for the content. And a `Span` can be created in a few
+different ways.
+
+1. using `Span::raw`:
+
+```rust
+fn ui(_app: &App, f: &mut Frame<'_>) {
+    let span = Span::raw("This is text that is not styled");
+    // -- snip ---
+}
+```
+
+2. using `Span::styled`:
+
+```rust
+fn ui(_app: &App, f: &mut Frame<'_>) {
+    let span = Span::styled("This is text that will be yellow", Style::default().fg(Color::Yellow));
+    // -- snip ---
+}
+```
+
+3. using the `Stylize` trait:
+
+```rust
+fn ui(_app: &App, f: &mut Frame<'_>) {
+    let span = "This is text that will be yellow".yellow();
+    // -- snip ---
+}
+```
+
+`Span`s are the building blocks to build styled content that is text that is represented
+horizontally.
+
+## `Line`
+
+The next building block that we are going to talk about is a `Line`. A `Line` represents a single
+"line" of text. The `Line` struct is essentially a `Vec<Span>`.
+
+Each Line comprises of multiple `Span` objects which allow for varied styling within a single line.
```

Comment:
"Each Line comprises multiple" or "Each `Line` consists of multiple." Small grammar nitpick.

Also, maybe put a comma after "`Span` objects" to separate the sentence into two, and change "allow" -> "allows", which I think reads better.

Reply:
Fixed in https://github.com/ratatui-org/ratatui-book/pull/72/commits/91e64b3162aba891f361c13105eb992c8fad86dc.